index.html

```diff
@@ -45,6 +45,110 @@ <h3 id="tickets">Tickets</h3>
         </a>
       </aside>
 
+      <section id="schedule">
+        <aside class="left">
+          <h3>Schedule</h3>
+          <table class="schedule">
+            <thead>
+              <tr>
+                <th>Time</th>
+                <th>Action</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>08:45 - 09:00</td>
+                <td>Get in</td>
+              </tr>
+              <tr>
+                <td>09:00 - 09:30</td>
+                <td><i>Breakfast</i></td>
+              </tr>
+              <tr>
+                <td>09:30 - 09:45</td>
+                <td><strong>Welcome to the CouchDB Day</strong></td>
+              </tr>
+              <tr>
+                <td>09:45 - 10:15</td>
+                <td><strong>Keynote by Jan Lehnardt</strong></td>
+              </tr>
+              <tr>
+                <td>10:15 - 10:30</td>
+                <td>Find your group you want to work in</td>
```

Comment:
Proposals for workshops - propose your topic in around 1min


idee dahinter ist das jeder der einen workshop geben mag oder an einem workshop interessiert ist, das ganze kurz vorstellt. die leute die mitmachen wollen, können sie dann, wenn alle durch sind, zu der person stellen. das sollten wir eventuell noch in die mail schreiben.